Client/Frontend/Browser/ZoomPageBar.swift

```diff
@@ -11,69 +11,82 @@ protocol ZoomPageBarDelegate: AnyObject {
 }
 
 class ZoomPageBar: UIView {
```

Comment:
The view is not reacting correctly to theme changes, it should implement `ThemeApplicable` and update the theme on `applyTheme(theme: Theme)`
Ex: Changing from light to dark mode
![Simulator Screen Shot - iPhone 13 Pro - 2023-03-24 at 09 58 25](https://user-images.githubusercontent.com/18538058/227541829-f2b44ec0-e100-49fb-96be-d53a589a3036.png)